### PR TITLE
Remove default config transfer limits

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

There were default transfer limits in config.py instead of a config file. Since sometimes we want no transfer limits, we don't want them here! Removes default configuration transfer limits.  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[Remove default config transfer limits](https://trello.com/c/VfXpRd9m/843-remove-default-config-transfer-limits)

File: config.py [hunk 1]
```diff
@@ -118,6 +118,7 @@
 
 PROGRAM_NAME        = config_parser['APP']['PROGRAM_NAME']
 
+DISABLE_LIMITS = config_parser['APP'].getboolean('DISABLE_LIMITS', True)
```
Comment (tristanhcole) [inline config.py:121]: I think this should be `False` by default

File: config_files/public/docker_test_config.ini [hunk 2]
```diff
@@ -25,6 +25,7 @@ third_party_sync_error_detection_interval    = 120
 third_party_sync_error_detection_grace_period= 1800
 chains                                       = ETHEREUM
 default_chain                                = ETHEREUM
+disable_limits                               = false
```
Comment (tristanhcole) [inline config_files/public/docker_test_config.ini:28]: can we also add to `local_config.ini`?

**Review by tristanhcole — APPROVED** [review]
Tested & good to go

**Pull request merged**